sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -170,6 +170,9 @@ kernel_pid_t gnrc_sixlowpan_nd_next_hop_l2addr(uint8_t *l2addr, uint8_t *l2addr_
         *l2addr_len = sizeof(eui64_t);
         memcpy(l2addr, &next_hop->u8[8], sizeof(eui64_t));
         _revert_iid(l2addr);
+/* in case of a border router there is no sensible way for address resolution
+ * if the interface is not given */
+#ifndef MODULE_GNRC_SIXLOWPAN_ND_BORDER_ROUTER
```

Comment:
True. I had there at first, but then for some mysterious reason moved it down. Will revert.